wordpress-plugin/gk-block-mcp/tests/Block/BlockNormalizerTest.php

```
@@ -330,6 +330,400 @@ public function test_engine_applies_any_registered_normalizer() {
+		);
+
+		$this->assertNotWPError( $result );
+		$this->assert_stored_list_baked( $post_id );
+	}
+
+	/**
+	 * create_post is the sibling write funnel (Post_Manager serializes blocks
+	 * directly, bypassing save_blocks). It must bake an invalid list too.
+	 */
+	public function test_create_post_normalizes_invalid_list() {
+		wp_set_current_user( self::factory()->user->create( array( 'role' => 'editor' ) ) );
```

PHPStan (2.2.7)

[error] Call to an undefined static method BlockNormalizerTest::factory().


(staticMethod.notFound)